sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/Models/DataFeedSourcesTests.cs

```diff
@@ -24,9 +24,11 @@ public class DataFeedSourcesTests : MockClientTestBase
             new object[] { new AzureCosmosDbDataFeedSource("secret", "mock", "mock", "mock"), "connectionString" },
             new object[] { new AzureDataExplorerDataFeedSource("secret", "mock"), "connectionString" },
             new object[] { new AzureDataLakeStorageGen2DataFeedSource("mock", "secret", "mock", "mock", "mock"), "accountKey" },
+            new object[] { new AzureEventHubsDataFeedSource("secret", "mock"), "connectionString" },
```

Comment:
I loved the fact that the changes to the non-live tests were minimum. I'll think about if it's possible to do the same for our live tests, but only when I have time. Tried to think a bit, but there are more scenarios.